tests/case/__snapshots__/jsfmt.spec.js.snap

```diff
@@ -291,7 +291,7 @@ final class MyClass extends Bar implements iTemplate
 
     private $private = 'Private';
 
-    public $foo = 'Bar';
+    var $foo = 'Bar';
```

Comment:
Big bug :smile: Original code use `var`, we change `var` on `public`

Reply:
:smile: